backend/danswer/background/connector_deletion.py

```diff
@@ -108,48 +116,6 @@ def delete_connector_credential_pair_batch(
             db_session.commit()
 
 
-def cleanup_synced_entities(
```

Comment:
No longer do any additional syncing after the fact—by the time we've gone through all the batches everything in vespa should be correct